### PR TITLE
cstor-base containerization

### PR DESCRIPTION
**Pull request by gkGaneshR**
1. Why is this change necessary ?
  The docker image for cstor base container needs to be created

2. How does this change address the issue ?
   The corresponding dockerfile and travis yaml are uploaded

3. How to verify this change ?
   docker pull openebs/cstor-base:ci

Signed-off-by: gkGaneshR <gkganesh126@gmail.com>

### CODE REVIEW

File: cstor-container [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+git checkout spl-0.7-release
+sh autogen.sh
+./configure
+make
+
+cd ../zfs
+pwd
+ls
+sh autogen.sh
+./configure --enable-uzfs=yes --with-jemalloc
```
Comment (gila) [inline cstor-container]: add --with-config=user

File: cstor-container [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+echo "listing zfs/cmd/*"
+ls zfs/cmd/zrepl/
+ls zfs/cmd/zpool/
+ls zfs/cmd/zfs/
+
+ls zfs/cmd/
+
+ls
+echo "Build Completed"
```
Comment (gila) [inline cstor-container]: Why all the "ls" commands?

File: cstor-container (hunk 1 above)
Comment (gkGaneshR) [reply cstor-container]: will add in the next commit

Comment (gkGaneshR) [reply cstor-container]: --with-config=user -> user is a static string right? or is it an env variable?

**Comment by gkGaneshR** [comment]
Will close this and raise a new pr, coz., of some travis yml issue.

**Pull request closed without merging**